docs/deploy-rbd.md

```diff
@@ -196,6 +198,28 @@ The Helm chart is located in `charts/ceph-csi-rbd`.
 
 [See the Helm chart readme for installation instructions.](../charts/ceph-csi-rbd/README.md)
 
+## Read Affinity using crush locations for RBD volumes
+
+Ceph CSI supports mapping RBD volumes with krbd options
+`"read_from_replica=localize,crush_location=type1:value1|type2:value2"` to
+optimize for reads from the most local serving OSD (OSD assigned in the
+same ceph CRUSH map location i.e.,region,zone,rack,etc).
```

Comment:
The CRUSH location doesn't have to be the _same_, the proximity gets calculated:
```suggestion
Ceph CSI supports mapping RBD volumes with krbd options
`"read_from_replica=localize,crush_location=type1:value1|type2:value2"` to
allow serving reads from the most local OSD (according to OSD locations as
defined in the CRUSH map).
```